handycon.py

```diff
@@ -7,18 +7,19 @@
 # presses that Steam understands.
 
 import asyncio
-import logging
-import os
-import signal
-import subprocess
-import warnings
 import configparser
-
 from evdev import InputDevice, InputEvent, UInput, ecodes as e, list_devices, ff
-from pathlib import PurePath as p
+import logging
```

Comment:
I prefer to have all the full imports at the top, then partial libarary imports, then local imports. This helps readability a lot when you have lots of imports.